pkg/workceptor/command.go

```diff
@@ -209,6 +209,14 @@ loop:
 			MainInstance.nc.GetLogger().Error("Error updating status file %s: %s", statusFilename, err)
 		}
 	}
+	err = stdin.Close()
```

Comment:
Im a bit worried about these two for longer running jobs, 
Have you tested this with either
A job lasting longer than 5 min
Using defer
😄